vg/vgtex/canvas.go

```diff
@@ -0,0 +1,272 @@
+// Copyright 2016 The gonum Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package vgtex provides a vg.Canvas implementation for LaTeX, using the pgf LaTeX package.
+package vgtex
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"image/color"
+	"io"
+	"math"
+	"strings"
+
+	"github.com/gonum/plot/vg"
+)
+
+const (
+	pr      = 5 // precision to use when outputting float64s
+	rad2deg = 180 / math.Pi
```

Comment:
I'm not completely happy with this name. Ideally you want something that means "degrees per radian" but is not as long.